docs/paper/joss.md

```diff
@@ -0,0 +1,263 @@
+$$
+\begin{align}
+    \widetilde{\rho}\frac{\partial \mathbf{v}}{\partial \psi} + \nabla\cdot\mathbf{\tau} - \nabla p = \mathbf{f}  \\
+    \frac{1}{\widetilde{K}}\frac{\partial p}{\partial \psi} + \nabla\cdot\mathbf{v} = \beta \frac{\partial p}{\partial t} + \alpha \frac{\partial T}{\partial t} \\
+    \frac{1}{2\widetilde{G}} \frac{\partial\mathbf{\tau}}{\partial\psi} + \frac{\mathbf{\tau}}{2\eta} = \dot{\mathbf{\varepsilon}} \\
+\end{align}
+$$
+
+where the wide tilde denotes the effective damping coefficients and $\psi$ is the pseudo-time step. These are defined as in [@Raess2022]:
+$$
+\begin{align}
+    \widetilde{\rho} = Re\frac{\eta}{\widetilde{V}L}, \qquad
+    \widetilde{G} = \frac{\widetilde{\rho} \widetilde{V}^2}{r+2}, \qquad
+    \widetilde{K} = r \widetilde{G}
+\end{align}
+$$
+with
+$$
+\begin{align}
+    \widetilde{V} = \sqrt{ \frac{\widetilde{K} +2\widetilde{G}}{\widetilde{\rho}}}, \qquad
+    r = \frac{\widetilde{K}}{\widetilde{G}}, \qquad
+    Re = \frac{\widetilde{\rho}\widetilde{V}L}{\eta}
+\end{align}
+$$
+where the P-wave $\widetilde{V} = V_p = \widetilde{C} \frac{\Delta x}{\Delta \tau}$ is the characteristic velocity scale, and $Re$ is the Reynolds number. We refer the reader to [@Raess2022] for further details on these damping coefficients. Similarly to the Stokes equations, the PT heat diffusion equation is given by:
+$$
+\begin{align}
+    \widetilde{\rho}\frac{\partial T}{\partial \psi} + \rho C_p \frac{\partial T}{\partial t} = -\nabla q \\
+    \widetilde{\theta}\frac{\partial q}{\partial \psi} + q  = -k\nabla T \\
+\end{align}
+$$
+
+where
+$$
+\begin{align}
+  \widetilde{\rho} = R_e \frac{k}{\widetilde{V} L}, \qquad
+  \theta_r = \frac{L}{R_e \widetilde{V}}, \qquad
+  Re = \pi + \sqrt{\pi^2 + \frac{\rho C_p}{k}L^2}
+\end{align}
+$$
+
+The APT equations are solved by discretizing the pseudo-time derivatives, either explicitly or implicitly, and iterating until the residual form of the equations is below a given tolerance. This means that the wave component of the equation has completely damped out and the pseudo-time derivative is approximately zero. In JustRelax.jl, the APT equations are discretized implicitly, as this yields better convergence rates.
+
+### Advection
+
+The advection equation in a decoupled manner using a Particle-in-Cell (PiC) method to advect the temperature, composition, or any other information carried by the particles. This method is particularly well-suited for this task because it can accurately handle the advection of these fields @dominguez2024, and it is extensively used to simulate global and regional scale geodynamic processes, e.g. StaggYY [@stagyy], LaMEM [@lamem], I3ELVIS [@i3elvis], ASPECT [@aspect], amongst others. This is implemented in the publicly available [JustPIC.jl](https://github.com/JuliaGeodynamics/JustPIC.jl) package, where different time integrators and velocity interpolation schemes are available.
```

Comment:
```suggestion
The advection equation is solved in a decoupled manner using a Particle-in-Cell (PiC) method to advect the temperature, composition, or any other information carried by the particles. The PiC method is particularly well-suited for this task because it can accurately handle the advection of these fields @dominguez2024, and it is extensively used to simulate global and regional scale geodynamic processes, e.g. StaggYY [@stagyy], LaMEM [@lamem], I3ELVIS [@i3elvis], ASPECT [@aspect], amongst others. PiC advection is implemented in the publicly available [JustPIC.jl](https://github.com/JuliaGeodynamics/JustPIC.jl) package, where different time integrators and velocity interpolation schemes are available.
```